### ISSUE TITLE
Test fastq option w/ long reads throws Error in library(ggplot2) : there is no package called ‘ggplot2’

### ISSUE THREAD

**Issue by hoelzer** [replikation/What_the_Phage#25]
I tested the wf again with some nanopore long reads using the new ``--fastq`` input option:

````
nextflow run phage.nf --fastq ../virify/results/SRR8811964_1.fastq/SRR8811964_1.fastq.unclassified.fastq --databases /homes/mhoelzer/data/nextflow-databases/ --workdir /homes/mhoelzer/data/nextflow-work-mhoelzer --cachedir /hps/nobackup2/singularity/mhoelzer/ -profile lsf
````

Seems to work very well until the final step:

````
[14/e74e2b] process > read_validation_wf:removeSmallReads (13) [100%] 13 of 13, cached: 7 ✔
[0e/1c10e1] process > read_validation_wf:fastqTofasta (4)      [100%] 13 of 13 ✔
[ac/08b407] process > metaphinder_wf:metaphinder (2)           [100%] 13 of 13 ✔
[cc/b561f6] process > metaphinder_wf:filter_metaphinder (1)    [100%] 1 of 1 ✔
[ac/7f5adf] process > virfinder_wf:virfinder (9)               [100%] 13 of 13 ✔
[8c/010786] process > virfinder_wf:filter_virfinder (1)        [100%] 1 of 1 ✔
[8d/9e7b2b] process > pprmeta_wf:pprmeta (8)                   [100%] 13 of 13 ✔
[be/254b31] process > pprmeta_wf:filter_PPRmeta (1)            [100%] 1 of 1 ✔
[17/009afe] process > parse_reads (1)                          [100%] 1 of 1 ✔
[ef/634e2e] process > r_plot_reads (1)                         [100%] 1 of 1, failed: 1 ✘
````

that errors with:

````
Error executing process > 'r_plot_reads (1)'

Caused by:
  Process `r_plot_reads (1)` terminated with an error exit status (1)

Command executed:

  #!/usr/bin/Rscript

  library(ggplot2)

  inputdata <- read.table("SRR8811964_1.fastq.unclassified_summary.csv", header = TRUE, sep = ";")

  pdf("phage-distribution.pdf", height = 6, width = 10)
    ggplot(data=inputdata, aes(x=type, y=amount)) +
    geom_bar(stat="identity") +
    theme(legend.position = "none") +
    coord_flip()
  dev.off()

Command exit status:
  1

Command output:
  (empty)

Command error:
  Error in library(ggplot2) : there is no package called ‘ggplot2’
  Execution halted
````

**Comment by replikation** [comment]
this sounds like a singularity issue? because ggplot is inside the docker. or i have not updated your config. ill check that. But yay it run through till the plotting :D

**Comment by replikation** [comment]
@hoelzer yeah its the same ggplot container. @Stormrider935 can you reproduce the error?

**Comment by Stormrider935** [comment]
* This is what i got:

`nextflow run replikation/What_the_Phage --fastq 'What_the_Phage/test-data/*.fastq*' --cores 20 --workdir /home/mike/install_software/`

* Error-msg

>Error executing process > 'r_plot_reads (1)'

>Caused by:
  Process `r_plot_reads (1)` terminated with an error exit status (127)

>executor >  local (10)
[f2/10ab41] process > read_validation_wf:removeSmallReads (1)   [100%] 1 of 1 ✔
[48/9e882e] process > read_validation_wf:fastqTofasta (1)       [100%] 1 of 1 ✔
[35/b839de] process > metaphinder_wf:metaphinder (1)            [100%] 1 of 1 ✔
[93/c4a22c] process > metaphinder_wf:filter_metaphinder (1)     [100%] 1 of 1 ✔
[25/32bdaf] process > virfinder_wf:virfinder (1)                [100%] 1 of 1 ✔
[bd/468987] process > virfinder_wf:filter_virfinder (1)         [100%] 1 of 1 ✔
[skipped  ] process > ppr_dependecies:ppr_download_dependencies [100%] 1 of 1, stored: 1 ✔
[eb/d719e4] process > pprmeta_wf:pprmeta (1)                    [100%] 1 of 1 ✔
[60/6d615d] process > pprmeta_wf:filter_PPRmeta (1)             [100%] 1 of 1 ✔
[e6/a4adb1] process > parse_reads (1)                           [100%] 1 of 1 ✔
[7c/567ff0] process > r_plot_reads (1)                          [100%] 1 of 1, failed: 1 ✘
Error executing process > 'r_plot_reads (1)'

>Caused by:
  Process `r_plot_reads (1)` terminated with an error exit status (127)

>Command executed:

>#!/usr/bin/Rscript

>library(ggplot2)

>inputdata <- read.table("fastq_set.fastq_summary.csv", header = TRUE, sep = ";")

>pdf("phage-distribution.pdf", height = 6, width = 10)
    ggplot(data=inputdata, aes(x=type, y=amount)) +
    geom_bar(stat="identity") +
    theme(legend.position = "none") +
    coord_flip()
  dev.off()

>Command exit status:
  127

>Command output:
  (empty)

>Command error:
  .command.run: line 56: /usr/bin/Rscript: No such file or directory

>Work dir:
  /home/mike/install_software/7c/567ff0be4cd0d6c7ae1025739877b3

**Comment by replikation** [comment]
so you can not reproduce the error.... -.-
this looks abit like it was using the wrong docker somehow?

**Comment by replikation** [comment]
ah i found the error.

**Comment by replikation** [comment]
@hoelzer @Stormrider935 should be fixed in a99a759

**Comment by replikation** [comment]
please close if working, thanks
